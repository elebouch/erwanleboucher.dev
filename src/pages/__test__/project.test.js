import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import Projects from "../projects"

describe("Projects", () => {
  it("renders correctly", () => {
    const { container } = render(<Projects />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
