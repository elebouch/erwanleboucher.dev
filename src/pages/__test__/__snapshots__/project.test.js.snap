// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders correctly 1`] = `
"<article><div class=\\"sc-AxirZ dqAfRA\\"><div class=\\"sc-AxjAm lnFVSZ\\"></div><span class=\\"sc-AxiKw eOeWbt\\">Projects</span><div class=\\"sc-AxhCb xKost\\"><p class=\\"sc-AxhUy HUQrh\\"><a href=\\"https://github.com/eleboucher/erwanleboucher.dev\\" class=\\"sc-AxheI bGBkVa\\">erwanleboucher.dev</a></p><p class=\\"sc-AxgMl jeRSjJ\\">stars: 3 | forks:
          0</p><p>Personal Website using the GatsbyJS. Minimalistic website to display my journey as a software developer.</p><span>(JavaScript, CSS, HTML, TypeScript)</span></div><a href=\\"https://github.com/eleboucher\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\" aria-label=\\"Github\\" style=\\"margin-top: 30px; font-weight: bold;\\" class=\\"sc-AxheI bGBkVa\\">More on Github</a></div></article>"
`;
